src/cron/arweave/arweave.rs

```diff
@@ -108,7 +108,7 @@ pub enum ArweaveProtocol {
 #[derive(Clone)]
 pub struct Arweave {
     pub host: String,
-    pub port: i32,
+    pub port: i64,
```

Comment:
I suppose changing this was an accident. Actually, I would define the port as `u16`.